### PR TITLE
Add Heart BioAmp Candy in "Hardware" section

### PR DESCRIPTION
**Pull request by bhawnasehgal**
Hey @lorforlinux 

I have made the following updates in Heart BioAmp Candy documentation:

- Added content including description, features, and how to use the sensor
- Added Candy images, connection diagrams, and snapshot for ECG signals recording using the sensor.

Please review and merge.

### CODE REVIEW

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+Heart BioAmp Candy is a candy-sized sensor for recording ECG signals easily. It is perfect for students and researchers 
+as it offers reliable and accurate ECG recordings, making heart monitoring accessible and convenient for educational and 
+research purposes.
+
+.. figure:: media/heart-bioamp-candy-front.*
+    :width: 100%
+    :align: center
+    :alt: Heart BioAmp Candy
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:17]:
```suggestion

.. figure:: media/heart-bioamp-candy-front.*
    :width: 100%
    :align: center
    :alt: Heart BioAmp Candy

   Heart BioAmp Candy
```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+Hardware
+*************
+
+.. figure:: media/candy-back.*
+    :width: 800
+    :align: center
+    :alt: Heart BioAmp Candy
+
+    Assembled PCB - Back
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:50]:
```suggestion
.. figure:: media/candy-back.*
    :width: 800
    :align: center
    :alt: Heart BioAmp Candy

    Assembled PCB Render - Back
```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+*********
+
+Heart BioAmp Candy is a candy-sized sensor for recording ECG signals easily. It is perfect for students and researchers 
+as it offers reliable and accurate ECG recordings, making heart monitoring accessible and convenient for educational and 
+research purposes.
+
+.. figure:: media/heart-bioamp-candy-front.*
+    :width: 100%
+    :align: center
+    :alt: Heart BioAmp Candy
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:16]:
```suggestion
.. figure:: media/candy-angled.*
    :width: 100%
    :align: center
    :alt: Heart BioAmp Candy
```

File: hardware/bioamp/heart-bioamp-candy/index.rst (hunk 2 above)
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:50]:
```suggestion

.. figure:: media/-heart-bioamp-candy-front.*
    :width: 800
    :align: center
    :alt: Heart BioAmp Candy

    Assembled PCB Render - Front
    
.. figure:: media/candy-back.*
    :width: 800
    :align: center
    :alt: Heart BioAmp Candy

    Assembled PCB Render - Back
```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
++------------------------------+-----+
+| Heart BioAmp Band (ECG Band) | 1   |
++------------------------------+-----+
+| Gel electrodes               | 12  |
++------------------------------+-----+
+
+.. figure:: media/candy-kit-contents.*
+    :align: center
+    :alt: kit contents
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:86]:
```suggestion

.. figure:: media/candy-kit-contents.*
    :align: center
    :alt: kit contents
    
    Kit contents
```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+Software requirements
+**********************
+
+1. **Uploading the code:** Before you start using the kit, please download `Arduino IDE <https://www.arduino.cc/en/software>`_. Using this you'll be able to upload the Chords Arduino Firmware on your development board.
+2. **Visualizing the signals:** After uploading the firmware, you can use `Chords <https://chords.upsidedownlabs.tech/>`_, an open-source web application developed by us to record and visualize real-time biopotential signals (ECG, EMG, EEG, EOG).
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:92]:
```suggestion
Software requirements
**********************

1. **Uploading the code:** Before you start using the kit, please download `Arduino IDE <https://www.arduino.cc/en/software>`_. Using this you'll be able to upload the `Chords Arduino Firmware <https://github.com/upsidedownlabs/Chords-Arduino-Firmware>`_ on your development board.
2. **Visualizing the signals:** After uploading the firmware, you can use `Chords <https://chords.upsidedownlabs.tech/>`_, an open-source web application developed by us to record and visualize real-time biopotential signals (ECG, EMG, EEG, EOG).

```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+Using the kit
+****************
+
+Step 1: Connections with development board
+============================================
+
+Connect ``VCC`` to either ``5V`` or ``3.3V``, ``GND`` to ``GND``, and ``OUT`` to ``Analog pin A0`` of your development board via jumper cables provided in the kit.
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:99]: Maybe we should add a table here

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+.. figure:: media/development-board-connections.*
+    :align: center
+    :width: 100%
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:104]:
```suggestion
.. figure:: media/development-board-connections.*
    :align: center
    :width: 100%
    :alt: Heart BioAmp Candy - Development board connection
    
    Heart BioAmp Candy - Development board connection
```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+Step 2: Connecting electrode cable
+========================================
+
+Connect the BioAmp cable to Heart BioAmp Candy by inserting the cable end in the JST PH connector as shown.
+
+.. figure:: media/electrode-cable-connections.*
+    :width: 100%
+    :align: center
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:117]:
```suggestion
.. figure:: media/electrode-cable-connections.*
    :width: 100%
    :align: center
    :alt: Heart BioAmp Candy - Electrode cable connections
    
    Heart BioAmp Candy - Electrode cable connections
```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
++++++++++++++++++++++++++++
+
+1. Connect the BioAmp cable to gel electrodes,
+2. Peel the plastic backing from electrodes
+3. Place the IN+,IN- and REF as shown in the connection diagram.
+
+.. figure:: media/ecg-using-gel-electrodes.*
+    :align: center
+    :alt: Using Heart BioAmp Candy to measure ECG with gel electrodes
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:143]:
```suggestion

.. figure:: media/ecg-using-gel-electrodes.*
    :align: center
    :alt: Using Heart BioAmp Candy to measure ECG with gel electrodes
    
    Using Heart BioAmp Candy to measure ECG with gel electrodes

```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+Using Heart BioAmp Band:
+++++++++++++++++++++++++++
+
+1. Snap the IN- cable on the left side, IN+ in the middle, and REF on the far side of the band.
+2. Now put a small drop of electrode gel between the skin and metallic part of BioAmp cable to get the best results.
+
+.. figure:: media/ecg-using-band.*
+    :align: center
+    :alt: Using Heart BioAmp Candy to measure ECG with Heart BioAmp Band
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:153]:
```suggestion

.. figure:: media/ecg-using-band.*
    :align: center
    :alt: Using Heart BioAmp Candy to measure ECG with Heart BioAmp Band
    
    Using Heart BioAmp Candy to measure ECG with Heart BioAmp Band

```

File: hardware/bioamp/heart-bioamp-candy/index.rst [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+Uploading the firmware
+------------------------
+
+Connect the development board to your laptop using a compatible USB cable. Copy paste the :fab:`github;pst-color-primary` `Chords Arduino Firmware <https://github.com/upsidedownlabs/Chords-Arduino-Firmware>`_ in Arduino IDE.
+
+.. figure:: media/chords-arduino-firmware.*
+    :align: center
+    :alt: Chords Arduino firmware
+
```
Comment (lorforlinux) [inline hardware/bioamp/heart-bioamp-candy/index.rst:164]:
```suggestion

.. figure:: media/chords-arduino-firmware.*
    :align: center
    :alt: Chords Arduino firmware
    
    Chords Arduino firmware

```

**Review by lorforlinux — COMMENTED** [review]
@bhawnasehgal I have added some comments, please check and resolve those so that I can merge this.

File: hardware/bioamp/heart-bioamp-candy/index.rst (hunk 6 above)
Comment (bhawnasehgal) [reply hardware/bioamp/heart-bioamp-candy/index.rst:107]: Agreed, adding the table

**Comment by bhawnasehgal** [comment]
@lorforlinux Thankyou for the inputs! I have resolved all the comments. You can go ahead and merge.

**Comment by lorforlinux** [comment]
@bhawnasehgal Thank you for your contributions, I am merging this now.

**Pull request merged**